book/src/app_validation.md

```diff
@@ -0,0 +1,26 @@
+# App Validation
+
+> **NOTE:** This chapter described the validation steps an application, using OpenMLS, has to perform for safe operation of the MLS protocol.
+>
+> **⚠️** This chapter is work in progress.
```

Comment:
Can you file an issue for this, or add a remark to an existing issue that we need to take care of this before the next release?

Reply:
👍🏻 Done